src/pages/guides/tutorials/grids-addon.md

```diff
@@ -0,0 +1,1321 @@
+			</div>
+			<div class="row">
+				<div class="column">
+					<sp-field-label for="cols" size="m">Columns</sp-field-label>
+					<sp-number-field id="cols"></sp-number-field>
+				</div>
+				<sp-swatch id="colsColorSwatch" class="color-well"></sp-swatch>
+				<input type="color" id="colsColorPicker" style="display: none;">
+			</div>
+		</div>
+		<div class="row">
+			<sp-slider label="Gutter" id="gutter" variant="filled" editable
+			           hide-stepper min="1" max="50" step="1"></sp-slider>
+		</div>
+		<sp-button-group horizontal>
+			<sp-button id="deleteGrid" disabled>Delete</sp-button>
+			<sp-button id="createGrid" disabled>Create</sp-button>
+		</sp-button-group>
+	</sp-theme>
+</body>
+```
+
+#### ui/index.js
+
+```js
+// All the SWC imports...
+
+import addOnUISdk from "https://new.express.adobe.com/static/add-on-sdk/sdk.js";
+
+addOnUISdk.ready.then(async () => {
+  console.log("addOnUISdk is ready for use.");
+
+  // Get the Authoring Sandbox.
+  const { runtime } = addOnUISdk.instance;
+  const scriptApi = await runtime.apiProxy("script");
+
+  // Input fields -------------------------------------------
+  const rowsInput = document.getElementById("rows");
+  const colsInput = document.getElementById("cols");
+  const gutterInput = document.getElementById("gutter");
+
+  rowsInput.value = 4;
+  colsInput.value = 6;
+  gutterInput.value = 10;
+
+  // Color pickers ------------------------------------------
+  const colsColorPicker = document.getElementById("colsColorPicker");
+  const colsColorSwatch = document.getElementById("colsColorSwatch");
+  const rowsColorPicker = document.getElementById("rowsColorPicker");
+  const rowsColorSwatch = document.getElementById("rowsColorSwatch");
+
+  colsColorPicker.value = "#ffcccc";
+  colsColorSwatch.color = "#ffcccc";
+  rowsColorPicker.value = "#ccccff";
+  rowsColorSwatch.color = "#ccccff";
+
+  colsColorSwatch.addEventListener("click", function () {
+    colsColorPicker.click();
+  });
+  colsColorPicker.addEventListener("input", function (event) {
+    const selectedColor = event.target.value;
+    colsColorSwatch.setAttribute("color", selectedColor);
+  });
+
+  rowsColorSwatch.addEventListener("click", function () {
+    rowsColorPicker.click();
+  });
+  rowsColorPicker.addEventListener("input", function (event) {
+    const selectedColor = event.target.value;
+    rowsColorSwatch.setAttribute("color", selectedColor);
+  });
+
+  // CTA Buttons --------------------------------------------
+  const createGridBtn = document.getElementById("createGrid");
+  const deleteGridBtn = document.getElementById("deleteGrid");
+
+  deleteGridBtn.onclick = async (event) => {
+    await scriptApi.deleteGrid();
+  };
+
+  createGridBtn.onclick = async (event) => {
+    await scriptApi.addGrid({
+      columns: colsInput.value,
+      rows: rowsInput.value,
+      gutter: gutterInput.value,
+      columnColor: colsColorPicker.value,
+      rowColor: rowsColorPicker.value,
+    });
```

Comment:
Following on from previous comment — this could then enable the "delete" button" now that the Grid is created.

Reply:
Could consider that as a "part 2" maybe

Reply:
Done